src/v2/Apps/Collect/Routes/Collections/index.tsx

```diff
@@ -44,16 +43,18 @@ export const CollectionsApp: React.FC<CollectionsAppProps> = ({
           </Column>
         </GridColumns>
 
-        <Join separator={<Spacer mt={6} />}>
-          {sorted.map((category, i) => {
-            return (
-              <CollectionsCategoryFragmentContainer
-                key={category.name + i}
-                category={category}
-              />
-            )
-          })}
-        </Join>
+        <Box data-test="collections-list">
```

Comment:
Added the `Box` from palette here because `data-test` prop in the `Join` component doesn't appear in the DOM and looks like the `Box` component doesn't affect the appearance of the elements